framework/src/mfem/auxkernels/MFEMHermitianInnerProductAux.C

```diff
@@ -0,0 +1,88 @@
+//* This file is part of the MOOSE framework
```

Comment:
Can we rebase and reimplement both the inner and external products in light of recent changes to their real counterparts?